libby_coy/week_02/02_homework_02/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8">
+    <title>Jenn</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="font-awesome-4.6.3/css/font-awesome.min.css">
+
+
+  </head>
+
+  <body class="index-body">
+    <header>
+        <a href="./index.html"><h1 class="index"><i>JENN <span style="color: #BF6658" hover="#109675">SCHIFFER</span></i></h1></a>
```

Comment:
There's no need to use that leading `./` for `href` or `src` paths when the file is in the same folder; it's customary to just write `href="index.html"`.